### PR TITLE
doc(readme): explain this docker image

### LINKED ISSUE #2: Add README
We should add a simple README describing what this repo contains and how to use the image etc.

@philippfranke would be great if you could add this.

### PR DESCRIPTION
**Pull request by philippfranke**
Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# docker-golang-arm
\ No newline at end of file
+# golang-arm
+
+This Docker image is based on the official image `golang` and contains   cross compiler for arm.
```
Comment (jhedev) [inline README.md]: so many spaces o.O

Comment (jhedev) [inline README.md]: between contains and cross

Comment (philippfranke) [inline README.md]: :D :D

Comment (philippfranke) [inline README.md]: 😆

**Pull request merged**